binGraph.py

```diff
@@ -145,6 +145,7 @@ def get_graph_modules():
 parser.add_argument('--dpi', type=int, default=__figdpi__, metavar=__figdpi__, help='Figure dpi')
 parser.add_argument('--blob', action='store_true', default=__blob__, help='Do not intelligently parse certain file types. Treat all files as a binary blob. E.g. don\'t add PE entry point or section splitter to the graph')
 parser.add_argument('-v', '--verbose', action='store_true', help='Print debug information to stderr')
+parser.add_argument('-w', '--watermark', action='store_true', help='Add watermark to image files')
```

Comment:
I would like to keep the attribution where possible. I don't mind setting this --nowatermark and default is False with store_true - it is good to have an option to remove it, but not as a default. Graphs can be created standalone (by running the graph.py module) and these are generated without the watermark.